specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -1534,6 +1534,134 @@
+            "whatsapp"
+          ],
+          "x-ms-enum": {
+            "name": "AcsMessageChannelType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "whatsapp",
+                "description": "Updated messaged channel type is Whatsapp"
+              }
+            ]
+          }
+        },
+        "media": {
+          "description": "The messaged received media content",
+          "$ref": "#/definitions/AcsAdvancedMessageMediaContent",
+          "type": "object"
+        },
+        "error": {
+          "description": "The The messaged received context",
+          "$ref": "#/definitions/AcsAdvancedMessageContext",
+          "type": "object"
+        },
+        "button": {
+          "description": "The The messaged received button content",
+          "$ref": "#/definitions/AcsAdvancedMessageButtonContent",
+          "type": "object"
+        },
+        "interactive": {
+          "description": "The The messaged received interactive content",
+          "$ref": "#/definitions/AcsAdvancedMessageInteractiveContent",
+          "type": "object"
+        }
+      },
+      "x-ms-examples": {
+        "snapshotCreatedCloudEventsSchema": {
+          "$ref": "./examples/cloud-events-schema/advanced_message_received.json"
+        },
+        "snapshotCreatedEventGridSchema": {
+          "$ref": "./examples/event-grid-schema/advanced_message_received.json"
+        }
+      }
+    },
+    "AcsAdvancedMessageDeliveryStatusUpdatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.Communication.AdvancedMessageDeliveryStatusUpdated event.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AcsAdvancedMessageEventBaseProperties"
+        }
+      ],
+      "properties": {
+        "messageId": {
+          "description": "The message id",
+          "type": "string"
+        },
+        "status": {
+          "description": "The updated message status",
+          "type": "string"
+        },
+        "channelType": {
+          "description": "The updated message channel type",
+          "type": "string",
+          "enum": [
+            "whatsapp"
+          ],
+          "x-ms-enum": {
+            "name": "AcsMessageChannelType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "whatsapp",
+                "description": "Updated messaged channel type is Whatsapp"
+              }
+            ]
+          }
+        }
+      },
+      "x-ms-examples": {
+        "snapshotCreatedCloudEventsSchema": {
+          "$ref": "./examples/cloud-events-schema/advanced_message_delivery_status_updated.json"
+        },
+        "snapshotCreatedEventGridSchema": {
+          "$ref": "./examples/event-grid-schema/advanced_message_delivery_status_updated.json"
+        }
+      }
+    },
+    "AcsAdvancedMessageEventBaseProperties": {
```

Comment:
Should be AcsAdvancedMessageEventData

Reply:
this is a based property that needed for both AcsAdvancedMessageReceivedEventData, AcsAdvancedMessageDeliveryStatusUpdatedEventData

Reply:
Understood - it should still end with EventData and not include Base in the name.